carmin.yaml

```diff
@@ -3,126 +3,281 @@ info:
   version: '1.0.0'
   title: CARMIN a Common web API for Remote pipeline INvocation
   description: FLI/IAM REST API for exchanging data and remote calling pipelines.
-    
-  #termsOfService: http://helloreverb.com/terms/
-  #contact:
-    #name: Swagger API team
-    #email: foo@example.com
-    #url: http://swagger.io
-  #license:
-    #name: MIT
-    #url: http://opensource.org/licenses/MIT
-#host: petstore.swagger.io
-#basePath: /api
-security: 
-  - {basicAuth:[]}
-  - {apiKey:[]}
+  # license TODO ?
+  # termsOfService TODO ?
+  # contact TODO ?
 schemes:
-  - http
-  - https
+  # - TODO should we include http ?
```

Comment:
I believe that it is good to forbid http since API key can be stolen from the HTTP header by a network listener